### PR TITLE
feat: unsplash daily

### PR DESCRIPTION
**Pull request by Michael-Liendo**

### CODE REVIEW

**Comment by EstebanBorai** [comment]
@Michael-Liendo can you pull from head please?

**Comment by Michael-Liendo** [comment]
@EstebanBorai Ready?

File: client/src/routes/api/unsplash/+server.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
+import { createApi } from 'unsplash-js';
+import type { RequestEvent } from './$types';
+import { UNSPLASH_ACCESS_KEY } from '$env/static/private';
+
+export type Unsplash = {
```
Comment (EstebanBorai) [inline client/src/routes/api/unsplash/+server.ts:5]:
```suggestion
import { createApi } from 'unsplash-js';

import { UNSPLASH_ACCESS_KEY } from '$env/static/private';

import type { RequestEvent } from './$types';

export type Unsplash = {
```

Comment (EstebanBorai) [inline client/src/routes/api/unsplash/+server.ts:5]: @Michael-Liendo can we use types from the package straight?

File: client/src/routes/api/unsplash/+server.ts [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        },
+      };
+
+      event.cookies.set('UNSPLASH_DAILY_BACKGROUND', JSON.stringify(response), {
+        expires: today,
+        path: '/',
+        httpOnly: true,
+        sameSite: 'strict',
+        secure: process.env.NODE_ENV === 'production',
+      });
```
Comment (EstebanBorai) [inline client/src/routes/api/unsplash/+server.ts:66]: Why do we need this cookie? 🤔

Comment (Michael-Liendo) [reply client/src/routes/api/unsplash/+server.ts:66]: To persist the same image every day, saving it in the cookie to expire in 24h

**Review by EstebanBorai — APPROVED** [review]
LGTM! Good job @Michael-Liendo!

**Pull request merged**